discretize/mixins/mpl_mod.py

```diff
@@ -0,0 +1,2070 @@
+from __future__ import print_function
```

Comment:
This is quite a big PR. I think it would make to get rid of all `from __future__ import` and `from six import` at once, what do you think?

Reply:
I agree. I had intended on doing this, but it slipped my mind. Thanks for the reminder!